pkg/apis/messaging/v1alpha1/channel_types.go

```diff
@@ -46,11 +47,20 @@ type Channel struct {
 	Status ChannelStatus `json:"status,omitempty"`
 }
 
-// Check that Channel can be validated, can be defaulted, and has immutable fields.
-var _ apis.Validatable = (*Channel)(nil)
```

Comment:
Grouped these `var` statements, similar to be consistent with the pattern in majority of other files.